gossip/hooks.py

```diff
@@ -343,7 +344,8 @@ def define(hook_name, **kwargs):
     return returned
 
 
-def register(func=None, hook_name=None, token=None, tags=None, needs=None, provides=None, reentrant=True, toggles_on=None, toggles_off=None, priority=0):
+def register(func=None, hook_name=None, token=None, tags=None, needs=None, provides=None, reentrant=True,
```

Comment:
When we drop 2.x support, one of the first things we need to add is keyword-only to this mess :-)